docusaurus.config.js

```diff
@@ -47,21 +47,26 @@ const config = {
         //disableSwitch: true
       },
       navbar: {
-        title: 'ColourBox',
-
         logo: {
           alt: 'ColourBox Logo',
-          src: 'img/logo.svg',
+          src: 'img/colourbox_logo_label.png',
         },
 
         items: [
+          { to: '/blog', label: 'RESOURCES', position: 'right' },
           {
             type: 'doc',
             docId: 'intro',
-            position: 'left',
-            label: 'Getting started',
+            position: 'right',
+            label: 'FAQs',
+          },
+          { to: '/blog', label: 'CONTACT US', position: 'right' },
```

Comment:
Remove INCORRECT links please. Contact us doesn't go to BLOG Once we merge this becomes live, We DONOT want bad links to be indexed. You can comment the code out and Add the links later

Reply:
I have commented out the uncompleted buttons